addons/mail/models/discuss/discuss_channel_rtc_session.py

```diff
@@ -77,7 +108,7 @@ def _gc_inactive_sessions(self):
         self.search(self._inactive_rtc_session_domain()).unlink()
 
     def action_disconnect(self):
-        self.unlink()
+        self.with_context(notify_rtc_server=True).unlink()
```

Comment:
This is strange, do we really need flux specific here?

And if that's the case, why not do it directly in the action?